core/conf/demo/server_config.yaml

```diff
@@ -165,3 +168,24 @@ gpu_resource_config:
 #----------------------+------------------------------------------------------------+------------+-----------------+
 tracing_config:
   json_config_path:
+
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# WAL Config           | Description                                                | Type       | Default         |
+#----------------------+------------------------------------------------------------+------------+-----------------+
+# enable               | Switch of function WAL.                                    | Boolean    | false           |
```

Comment:
Whether to enable write-ahead logging (WAL) in Milvus. If WAL is enabled, Milvus writes all data changes to log files in advance before implementing data changes. WAL ensures the atomicity and durability for Milvus operations.

Reply:
Please apply this change to other files.